src/ScatterTab.tsx

```diff
@@ -154,9 +159,10 @@ const ScatterTab = (props: ScatterTabProps): JSX.Element => {
                 variant="contained"
                 color="primary"
```

Comment:
Should the "clear" button be red or something?

Reply:
Yes, I agree. I tried that in a previous version but it turned out kinda wried. We shall experiment with that in the next commit :)